src/averageAges.js

```diff
@@ -15,11 +18,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const manArr = century
+    ? people.filter(
+        (item) => item.sex === "m" && Math.ceil(item.died / 100) === century
+      )
+    : people.filter((item) => item.sex === "m");
```

Comment:
you can apply ternary operator inside filter method
```suggestion
  const manArr = people.filter(
    (item) => item.sex === "m" && (!century || Math.ceil(item.died / 100) === century)
  );
```